files/ko/web/http/basics_of_http/identifying_resources_on_the_web/index.md

```diff
@@ -58,7 +58,7 @@ urn:ietf:rfc:7230
 ### 스키마 또는 프로토콜
 
 - ![Protocol](mdn-url-protocol@x2.png)
-- : `http://`는 프로토콜로, 브라우저가 사용해야 하는 프로토콜을 나타냅니다. 일반적으로 프로토콜은 HTTP이거나 보안 버전인 HTTPS입니다. 웹은 이 두 가지 중 하나를 사용하지만, 브라우저는 메일 클라이언트를 여는 `mailto:` 또는 파일 전송을 처리하는 `ftp:`와 같은 다른 프로토콜을 처리하는 방법도 알고 있으므로 이러한 프로토콜을 보더라도 놀라지 않으셔도됩니다. 일반적인 스키마는 다음과 같습니다.
```

Comment:
These are supposed to be definition lists in this file, but the indentation is wrong. Same problem with the table and the rest below

Reply:
Thanks you! I revised it according to your review.